docs/modular-resources/advanced/mlmodel-design.md

```diff
@@ -0,0 +1,122 @@
+---
+title: "Design your ML Models for Vision"
+linkTitle: "Vision Service and Modular ML Model Design"
```

Comment:
The link title should be the same as the title or shorter - this is the one that goes in the left nav
```suggestion
linkTitle: "ML Model Design"
```

Reply:
Ah sounds good, done